packages/fluentui/react-northstar/src/components/Card/CardPreview.tsx

```diff
@@ -0,0 +1,92 @@
+import { Accessibility } from '@fluentui/accessibility';
+import { getElementType, getUnhandledProps, useAccessibility, useStyles, useTelemetry } from '@fluentui/react-bindings';
+import * as _ from 'lodash';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+import { FluentComponentStaticProps, ProviderContextPrepared, WithAsProp, withSafeTypeForAs } from '../../types';
+import { ChildrenComponentProps, commonPropTypes, createShorthandFactory, UIComponentProps } from '../../utils';
+
+export interface CardPreviewProps extends UIComponentProps, ChildrenComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   * */
+  accessibility?: Accessibility<never>;
+
+  /**
+   *
+   * If preview is in horizontal card
+   */
+
+  horizontal?: boolean;
```

Comment:
```suggestion
  /** If preview is in horizontal card. */
  horizontal?: boolean;
```

I also suggest to improve description there